app/.composer/composer.json

```diff
@@ -0,0 +1,11 @@
+{
+    "require": {
```

Comment:
I think this is not a lesser way for managing these versions compared to the ENV variables used previously